lib/modules.nix

```
@@ -256,6 +256,7 @@ rec {
 
       merged =
         let collected = collectModules
+          specialArgs.class or null
```

```suggestion
          (specialArgs.class or null)
```

Technically not necessary but this feels illegal!